cmd/gen-go-sample/testdata/sample_unary.want

```diff
@@ -28,6 +28,14 @@ foopb "path.to/pb/foo"
 
 // [START awesome_region]
 
+// Construct a complex request object,
+// send it to the server,
+// and inspect the response.
+//
+// theX: a single-line comment for an input parameter
```

Comment:
The colons are fine, I think. Maybe we have a preceding line saying `// Parameters:`? But I don't think that's strictly needed.